docs/can-canjs/canjs.md

```diff
@@ -112,22 +112,32 @@ p + ul {
 .single-col-wrapper {
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  margin: 60px 30px;
+  max-width: 1440px;
+  margin: 40px auto;
+  width: 100%;
+}
+.single-col-wrapper .content-wrapper {
+  margin-left: 30px;
+  margin-right: 30px;
 }
-.single-col-wrapper .btn {
-  margin-top: 45px;
+.single-col-wrapper .cp_embed_wrapper {
+  margin: 30px 0 0 0;
```

Comment:
The natural spacing seemed fine to me… was it not?

Reply:
Natural space was fine. Felt like it could use a bit more breathing room. I'm fine to remove it.